Complex Radio/presence.ts

```diff
@@ -0,0 +1,26 @@
+var presence = new Presence({ clientId: "705189441484095508" });
+
+var ssong, slisteners, spresenter;
+
+async function newStats() {
+    const data = await window
+        .fetch("https://radio.complexr.pw/api/nowplaying/1")
+        .then((res) => res.json());
+    ssong = data.now_playing.song.text;
+    spresenter = data.live.is_live ? "Presenter " + data.live.streamer_name : "AutoDJ";
+    slisteners = data.listeners.unique + " Listeners";
+}
+
+setInterval(newStats, 2500);
+newStats();

```

Comment:
This is supposed to be `await newStats()`

Reply:
Send the screenshot directly to github by copy and pasting it into a comment.

Reply:
done that still issue with the deepsync